include/oneapi/tbb/detail/_concurrent_skip_list.h

```diff
@@ -72,19 +72,11 @@ class skip_list_node {
     using pointer = typename value_allocator_traits::pointer;
     using const_pointer = typename value_allocator_traits::const_pointer;
 
-    skip_list_node( size_type levels, container_allocator_type& alloc )
-        : my_container_allocator(alloc), my_height(levels), my_index_number(0)
-    {
-        for (size_type l = 0; l < my_height; ++l) {
-            allocator_traits::construct(my_container_allocator, &get_atomic_next(l), nullptr);
-        }
-    }
+    skip_list_node( size_type levels )
+        : my_height(levels), my_index_number(0)
+    {}
 
-    ~skip_list_node() {
-        for (size_type l = 0; l < my_height; ++l) {
-            allocator_traits::destroy(my_container_allocator, &get_atomic_next(l));
-        }
-    }
+    ~skip_list_node() {}
```

Comment:
May be remove the declaration at all?

Reply:
It is not possible. Due to union member defaulted/compiler generated constructor is deleted.  (https://eel.is/c++draft/class.dtor#7)